core/domain/platform_parameter_list.py

```diff
@@ -31,8 +31,6 @@ class ParamNames(enum.Enum):
     CHECKPOINT_CELEBRATION = 'checkpoint_celebration'
     CONTRIBUTOR_DASHBOARD_ACCOMPLISHMENTS = (
         'contributor_dashboard_accomplishments')
-    ANDROID_BETA_LANDING_PAGE = 'android_beta_landing_page'
-    BLOG_PAGES = 'blog_pages'
```

Comment:
These shouldn't actually be removed--we need to retain the names of shipped parameters & feature flags indefinitely.

Reply:
Sure.